### DISCUSSION TITLE
errors on relative imports

### DISCUSSION THREAD

**Discussion by RamiKhawaly** [micropython/micropython#12868] [ESP32] [answered]
Hi, I'm new to micropython and i'm a bit lost. 
I am writing a small project that should use the micropython_ir library. I downloaded it to my project, but when I flash it and run it, it fails on the imports inside the micropython_ir files. the imports there are relative, how i can make Python recognize these imports?
I can't change all the imports also it's the wrong attitude. 
one more question related to using the libraries, is it the right way to download the source code and add it to my project or there is another way? i tryed using mpremote but it keeps saying that the packages does not found. 

I'm running on esp32 espressife board. 

Thanks

**Comment by jimmo** [comment, marked as answer]
Could you please provide examples of the mpremote commands you ran?

Rather than copying the files manually you can use mip (via mpremote) to install the IR libraries -- e.g. see 
https://github.com/peterhinch/micropython_ir/blob/master/TRANSMITTER.md#22-installation

**Reply by RamiKhawaly** [reply]
thanks for the help

**Comment by peterhinch** [comment]
Installation with `mpremote` will create the directory `/lib/ir_tx/` or `/lib/ir_rx/` depending on whether you want to transmit or receive. `/lib` is on the search path, so commands like
```py
from ir_tx.nec import NEC
```
will import the NEC transmitter driver `/lib/ir_tx/nec.py` (also  `/lib/ir_tx/__init__.py`).

**Reply by RamiKhawaly** [reply]
thanks for the help
